test/affinity_assistant_test.go

```diff
@@ -84,12 +81,13 @@ spec:
 		t.Fatalf("Error waiting for PipelineRun to run: %s", err)
 	}
 
-	// validate SS and PVC are created
-	ssName := pipelinerun.GetAffinityAssistantName("my-workspace", pr.Name)
-	ss, err := c.KubeClient.AppsV1().StatefulSets(namespace).Get(ctx, ssName, metav1.GetOptions{})
-	if err != nil {
-		t.Fatalf("Couldn't get expected Affinity Assistant StatefulSet %s, err: %s", ss, err)
+	// wait for PipelineRun to finish
+	t.Logf("Waiting for PipelineRun in namespace %s to finish", namespace)
+	if err := WaitForPipelineRunState(ctx, c, prName, timeout, PipelineRunSucceed(prName), "PipelineRunSucceeded", v1Version); err != nil {
```

Comment:
Would it make sense to remove the previous call to `WaitForPipelineRunState`, as it's now redundant?

Reply:
Yep, fixed!